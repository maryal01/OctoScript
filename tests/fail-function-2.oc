fn fail-function (int x) -> int {
    return x;
}

fail-function(12, 3);