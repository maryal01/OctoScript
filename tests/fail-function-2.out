Fatal error: exception Failure("Arguments-Parameters MisMatch")